src/components/forumComponents/showPost.js

```diff
@@ -0,0 +1,31 @@
+import { Paper } from "@mui/material";
+import { Typography } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import { useEffect } from "react";
+
+export const ShowPosts = ({title,text}) => {
```

Comment:
Nazwa sugeruje bardziej, że to jest jakaś funkcja która coś wyświtelta niż że jest to komponent renderujący post